stable/redis/templates/redis-slave-statefulset.yaml

```diff
@@ -76,6 +76,10 @@ spec:
         - /bin/bash
         - -c
         - |
+          {{- if (eq .Values.securityContext.runAsUser 0.0) }}
+          useradd redis
+          chown -R redis /data
```

Comment:
use {{ .Values.persistence.path }} instad of `/data`

Reply:
Applied.